hera_mc/tests/test_dataview.py

```diff
@@ -26,9 +27,25 @@ def test_ant_by_day(self):
         stop = cm_utils.get_astropytime(1184354600)
         interval = 1.0
         filename = None
-        x = dv.ants_by_day(start, stop, interval, filename, station_types_to_check='default', output_date_format='jd')
-        k0 = str(list(x.keys())[0]).split('.')[0]
-        self.assertTrue(k0 == '2457952')
+        n = dv.ants_by_day(start, stop, interval, filename, output_date_format='jd')
+        self.assertEqual(n, 3)
+        filename = os.path.expanduser('~/hera_mc_dataview_test.txt')
+        n = dv.ants_by_day(start, stop, interval, filename, output_date_format='iso')
+        self.assertEqual(n, 3)
+        os.remove(filename)
+
+    def test_connected_by_day(self):
+        dv = cm_dataview.Dataview(session=self.test_session)
+        start = cm_utils.get_astropytime(1184354550)
+        stop = cm_utils.get_astropytime(1184354600)
+        interval = 10.0
+        filename = None
+        n = dv.connected_by_day(start, stop, interval)
+        self.assertEqual(len(n.keys()), 1)
+        filename = os.path.expanduser('~/hera_mc_dataview_test.txt')
+        n = dv.connected_by_day(start, stop, interval, output=filename, output_date_format='iso')
+        self.assertEqual(n, None)
+        os.remove(filename)
```

Comment:
this file will not be removed if this test fails. Is that ok? We could handle this in the package setup/teardown (by making a folder to put test output into in setup and removing the folder in teardown) if we should.

Reply:
I tried teardown first, but it hung (remember, this happened in the past too).

Reply:
hmm, ok. I might try to move us in that direction in the future, but we can leave it alone for now. You could use a try/finally block to ensure the file is deleted if you want.